### ISSUE TITLE
Genome Browser: South America appears as a different data-type to the other regions

### ISSUE THREAD

**Issue by leehart** [wtchg-kwiatkowski/observatory-web#344]
Home > View genome > Add channel
1. Notice that "South America" has a different data-type icon to the others, i.e. a bar-chart rather than a line-graph
Add the channels "South America" and "West Africa" (to compare)
2. Notice that the tracks for "South America" and "West Africa" are plotted differently ("South America" is plotted with bars, whereas "West Africa" is plotted with points) and the channels have different options, legend, etc.
3. The settings for the columns shows no difference in `dataType`
```
- id: af_SAM
  dataType: Float
  name: South America
- id: af_WAF
  dataType: Float
  name: West Africa
```
I suspect/guess this might be caused by an internal algorithm for determining categorical-ness for number-type columns, maybe, with "South America" meeting the criteria but the data for other regions do not, maybe.

This issue is not present on the staging instance, so maybe my local version is out-of-date.

**Comment by benjeffery** [comment]
Yep, SAM has few enough samples that ratios of them don't count to more than 50 different ones. Fixable buy `isCategorical : false`

**Comment by leehart** [comment]
I gather testing with fewer variants locally explains the difference with staging.
(I suspect that might also invalidate this issue... but I suppose adding `isCategorical: false` to the settings would prevent this from cropping up again...)

**Comment by leehart** [comment]
Closing because `isCategorical: false` has been set for `af_SAM` for `pf_variants`, and this was only an issue for incomplete data in testing scenarios.
It might be worth noting that data are being categorized by default (effectively `isCategorical: true`), which is a "hidden feature", which might be potentially unexpected or unwanted.
Documentation currently states:
```
IsCategorical
Boolean. Instructs Panoptes to treat the property as a categorical variable. For example, a combo box with the possible states is automatically shown in queries for this property. Categorical properties are automatically indexed.
```
(I couldn't see any mention of the default behaviour.)

**Comment by leehart** [comment]
Reopening because the underlying issue might fester if left swept under the carpet.
